app/javascript/dashboard/components/widgets/WootWriter/Editor.vue

```diff
@@ -247,7 +247,7 @@ export default {
           },
         }),
         suggestionsPlugin({
-          matcher: triggerCharacters(':', 1), // Trigger after ':' and at least 1 characters
+          matcher: triggerCharacters(':', 2), // Trigger after ':' and at least 2 characters
```

Comment:
Why did we change this to 2?

Reply:
When adding emoticons like `:)`, the keyboard emoji selector remains active. To fix this, I set a minimum character limit of two. I also checked Discord, and their emoji selector similarly triggers after the second character.